docs/storage_snippets.py

```diff
@@ -220,6 +220,38 @@ def list_buckets(client, to_delete):
         to_delete.append(bucket)
 
 
+@snippet
+def policy_document(client, to_delete):
+    # [START policy_document]
+    bucket = client.bucket('my-bucket')
+    conditions = [
+        ['starts-with', '\$key', ''],
+        {'acl': 'public-read'}]
+
+    policy = bucket.generate_upload_policy(conditions)
+
+    # Generate an upload form using the form fields.
+    policy_fields = ''.join(
+        '<input type="hidden" name="{key}" value="{value}">'.format(
+            key=key, value=value)
+        for key, value in policy.items()
+    )
+
+    upload_form = (
```

Comment:
I am not passionate, but I would probably do the tried and true "make an empty list and append to it" for this, just because the list comprehension is a bit more in depth:

```python
policy_fields = []
for key, value in policy.items():
    policy_fields.append('<input type="hidden" name="{key}" value="{value}">'.format(key=key, value=value))
```

Apologies if that last line goes over 80 characters; harder to tell in GitHub (I bet it does).